frontend/packages/dev-console/src/utils/__tests__/__snapshots__/shared-submit-utils.spec.ts.snap

```diff
@@ -43,6 +43,7 @@ Object {
     "defaultAnnotations": Object {
       "app.openshift.io/vcs-ref": "master",
       "app.openshift.io/vcs-uri": "https://github.com/test/repo",
+      "openshift.io/generated-by": "OpenShiftWebConsole",
```

Comment:
@vikram-raj can we make the annotation consistent with other annotations? Something like `app.openshift.io/generated-by`. Or this is the ask to keep it like `openshift.io/generated-by`?

Reply:
@divyanshiGupta Agree with you. cc: @christianvogt

Reply:
I think this annotation was from the 3.x release. 
We've also been using it for a couple releases now as well so probably best to keep it.